frontend/src/metabase/visualizations/components/ScalarValue.jsx

```diff
@@ -27,30 +27,40 @@ const ScalarValue = ({ value, isFullscreen, isDashboard }) => (
   <h1 className="ScalarValue cursor-pointer text-brand-hover">{value}</h1>
 );
 
-export const ScalarTitle = ({ title, description, onClick }) => (
-  <div className="flex align-center full justify-center px2">
-    <h3
-      onClick={onClick}
-      className={cx(
-        "Scalar-title overflow-hidden fullscreen-normal-text fullscreen-night-text text-brand-hover",
-        {
-          "cursor-pointer": !!onClick,
-        },
+export const ScalarTitle = ({ title, description, onClick }) => {
+  const iconWidth = "24px";
```

Comment:
Typically I'd make this a top-level constant, like `const ICON_WIDTH = 24;`. Also you can leave off `px` when providing px values in React `style` props.